podo/PodoCount_code/xml_to_mask.py

```diff
@@ -4,6 +4,7 @@
 """
 import numpy as np
 import sys
+sys.path.append('..')

```

Comment:
this is not needed here. Please remove. We only add this in the path when we import within the repo, here we use external libraries.